src/components/page-header/page-header.css

```diff
@@ -0,0 +1,16 @@
+@import '../../styles/spacing.css';
+
+.container {
+  margin: var(--spacing-x-large);
```

Comment:
Adding padding to the body / page container has historically been avoided because of these kinds of designs:

<img width="1497" alt="Screen Shot 2020-05-19 at 11 31 05 AM" src="https://user-images.githubusercontent.com/841084/82364586-5102d000-99c4-11ea-98b5-872e2203ee73.png">

Notice that under the page header there is a line that extends from edge to edge. 

I think if we want to accommodate this use-case (which I believe to be reasonable) then we want to apply margin to top-level layout children elements. There will most likely be a tab component and it will extend from edge to edge but it will apply its own padding to align itself with the rest of the page layout elements.

Reply:
Fair enough. I'm worried about encouraging a pattern of elements being concerned with how they are laid out in their container, but considering this is a very walled-garden scenario, it's probably okay.